elixir/parameter/temp.py

```diff
@@ -6,37 +6,95 @@
 from torch.fx.immutable_collections import immutable_dict
 from torch.utils._pytree import tree_map
 
+from elixir import calc_buffer_size, gpu_dev
```

Comment:
`dev` usually stands for `development`. Don't use short names in general unless it is really commonly accepted.